File: Frontend/src/interfaces/v12/inventory/updateInventoryBalanceDto.ts
export interface UpdateInventoryBalanceDto {
  remark?: string | null;
  internalRemark?: string | null;
  agentPrice?: number | null;
  dealerPrice?: number | null;
  retailPrice?: number | null;
  costPrice?: number | null;
  locationId?: string | null;
}

File: Wms.Api/Dto/Inventory/UpdateInventoryBalanceDto.cs
namespace Wms.Api.Dto.Inventory
{
    public class UpdateInventoryBalanceDto
    {
        public string? Remark { get; set; }
        public string? InternalRemark { get; set; }
        public decimal? AgentPrice { get; set; }
        public decimal? DealerPrice { get; set; }
        public decimal? RetailPrice { get; set; }
        public decimal? CostPrice { get; set; }
        public Guid? LocationId { get; set; }
    }
}
